crates/cheatnet/src/cheatcodes/deploy.rs

```diff
@@ -70,6 +88,27 @@ impl CheatnetState {
             )),
         }
     }
+
+    pub fn deploy(
+        &mut self,
+        class_hash: &ClassHash,
+        calldata: &[Felt252],
+    ) -> Result<ContractAddress, CheatcodeError> {
+        let salt = self.get_salt();
+        let account_address = ContractAddress(patricia_key!(TEST_ACCOUNT_CONTRACT_ADDRESS));
+
+        let contract_address = calculate_contract_address(
+            salt,
+            *class_hash,
+            &Calldata(Arc::new(calldata.iter().map(felt_to_stark_felt).collect())),
+            account_address,
+        )
+        .unwrap();
```

Comment:
We might consider using `precalculate_address` cheatcode here in case we want to add some behavior there.